pkg/features/kube_features.go

```diff
@@ -772,6 +772,14 @@ const (
 	//
 	// Enables Leader Migration for kube-controller-manager and cloud-controller-manager
 	ControllerManagerLeaderMigration featuregate.Feature = "ControllerManagerLeaderMigration"
+
+	// owner: @jiahuif
+	// kep: http://kep.k8s.io/2887
+	// alpha: v1.23
+	//
+	// Enables populating "enum" field of OpenAPI schemas
+	// in the spec returned from kube-apiserver.
+	OpenAPIEnum featuregate.Feature = "OpenAPIEnum"
```

Comment:
I think "OpenAPIEnums" (plural) would sound better, do you agree?

Reply:
Yep. I will need to update the KEP too.

Reply:
Updated.